src/main/resources/default/templates/biz/jobs/job.html.pasta

```diff
@@ -44,11 +44,14 @@
         <div class="row">
             <div class="col-md-8">
                 <div class="row">
-                    <i:for type="sirius.biz.jobs.params.Parameter" var="param" items="job.getParameters()">
+                    <i:for type="sirius.biz.jobs.params.Parameter" var="param" items="job.getVisibleParameters(context)">
                         <i:dynamicInvoke template="@param.getTemplateName()" param="@param" context="@context"/>
```

Comment:
Wouldn't it be easier to create a new template file 'bla/bla/hidden.html.pasta' with
```
<input type="hidden" name="@param.getName()" value="@context.getOrDefault(param.getName(), null)"/>
```
and add some logic to the Parameter class that wraps `@param.getTemplateName()` and will return the 'bla/bla/hidden.html.pasta' if the parameter is hidden instead of the template name implemented in the parameter subclasses?

As an alternative, this could also be done using <i:if> right here in the template. 
These solutions would have the benefit of only iterating the parameter list once instead of three times.

Reply:
New template hidden.html.pasta returned by computeTemplateName which is now used by the pasta templates.